test/e2e/config/docker.yaml

```diff
@@ -21,6 +21,8 @@ images:
   loadBehavior: tryLoad
 - name: gcr.io/k8s-staging-cluster-api/test-extension-{ARCH}:dev
   loadBehavior: tryLoad
+- name: gcr.io/k8s-staging-cluster-api-helm/cluster-api-helm-controller-${ARCH}:v0.1.0-alpha.4
```

Comment:
Do we really need this? AFAICT none of the E2E tests are using the addon, right?